Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -459,6 +459,9 @@ public void setDebug() {
         enableTestMode();
     }
 
+    // Since play store referrer broadcast from google is few millisecond delayed, delay Branch init by calling this method for more accurate
+    // tracking and attribution. This will be only on the first time user open the app.
+    // Note : This method should be called in order to handle IA - Full app conversion deeplinking and attribution
```

Comment:
Let's add another note and please update with correctness:
`// Note: Recommend 1500 to capture more than 90% of the install referrer cases per our testing as of 4/2017`